sdk/identity/src/token_credentials/default_credentials.rs

```diff
@@ -98,20 +170,17 @@ impl TokenCredential for DefaultAzureCredentialEnum {
                     "error getting environment credential",
                 )
             }
-            DefaultAzureCredentialEnum::ManagedIdentity(credential) => {
-                // IMSD timeout is only limited to 1 second when used in DefaultAzureCredential
-                credential
-                    .get_token(scopes)
-                    .timeout(Duration::from_secs(1))
```

Comment:
I'm not sure if timeout is still needed. May be make it configurable. Is that settable in the HttpClient that is passed in?

Reply:
Reqwest support [timeout](https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.timeout) and connect_timeout.

Reply:
Leaving this the same. Not changing in this PR.